### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by pbarabe**
- Replace references to bitbucket with github (where this project is now hosted).
- Add docsify as a dev dependency.
- Replace protocol-relative URLs w/HTTPS.
- Remove Milo font

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,13 +12,16 @@
...
   "repository": {
     "type": "git",
-    "url": "git+https://bitbucket.com/uazgraduatecollege/developer-essentials"
+    "url": "git+https://github.com/uazgraduatecollege/developer-essentials"
   },
   "author": "",
   "license": "MIT",
   "bugs": {
-    "url": "https://bitbucket.com/uazgraduatecollege/developer-essentials/issues"
+    "url": "https://gighub.com/uazgraduatecollege/developer-essentials/issues"
```
Comment (uacate) [inline package.json:20]: Should this say github.com?

File: package.json [hunk 2]
```diff
@@ -12,13 +12,16 @@
...
   },
-  "homepage": "https://bitbucket.com/uazgraduatecollege/developer-essentials/#readme",
-  "dependencies": {}
+  "homepage": "https://gighub.com/uazgraduatecollege/developer-essentials/#readme",
```
Comment (uacate) [inline package.json:22]: @pbarabe  Should this say github.com?

File: package.json (hunk 1 above)
Comment (pbarabe) [reply package.json:20]: Good catch. Yes it should.

File: package.json (hunk 2 above)
Comment (pbarabe) [reply package.json:22]: Good catch. Yes it should. Wanna open an issue for these 2 typos?

Comment (pbarabe) [reply package.json:22]: I noticed that the license in package.json is incorrect, too.

`"license": "MIT",` should be `"license": "See license in file LICENSE,"`

**Pull request merged**